cleanlab/filter.py

```diff
@@ -298,16 +298,16 @@ class 0, 1, ..., K-1.
         # Create `prune_count_matrix` with the number of examples to remove in each class and
         # leave at least min_examples_per_class examples per class.
         # `prune_count_matrix` is transposed relative to the confident_joint.
-        prune_count_matrix = _keep_at_least_n_per_class(
+        prune_count_matrix:npt.NDArray[np.int_] = _keep_at_least_n_per_class(
```

Comment:
The prune_count_matrix changes the data type (from an array of int to float). Should I declare it as a float? Although the function here returns int explicitly.

Reply:
I'll have to take a better look later today.

Reply:
@elisno  did you have a chance to look at this?

Reply:
I don't see where the datatype should change in this function. I think this annotation is fine for now.